src/pybind/mgr/dashboard/controllers/host.py

```diff
@@ -394,6 +394,40 @@ def set(self, hostname: str, labels: List[str]):
         for label in add_labels:
             orch.hosts.add_label(hostname, label)
 
+    @RESTController.Resource('GET')
+    @raise_if_no_orchestrator([OrchFeature.HOST_MAINTENANCE_ENTER])
+    @handle_orchestrator_error('host')
+    def enter_maintenance(self, hostname: str, force=None):
```

Comment:
I was initially not too keen on the `force` flag, and I was going to suggest a `dry-run` flag instead, but then I checked how k8s does and they use both: [`force`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/) and [`dry-run`](https://kubernetes.io/docs/reference/using-api/api-concepts/#dry-run).

```suggestion
    def enter_maintenance(self, hostname: str, force:bool=None):
```

Reply:
done